react/week4/my-app/src/SearchInput.js

```diff
@@ -0,0 +1,11 @@
+import { useContext } from "react"
+import { UsersContext } from "./App"
+
+export default function Search() {
+    const {query, setQuery} = useContext(UsersContext)
+    return (
+        <>
+            <input type="search" value={query} onChange={e => setQuery(e.target.value)}></input>
+        </>
```

Comment:
You don't really need this wrapper...
```suggestion
          <input type="search" value={query} onChange={e => setQuery(e.target.value)} />
```